### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/e725c418a40443bed703b670d527d2b1e9d77d1a

### CODE REVIEW

File: en/firefox/installer-help.ftl [hunk 1]
```diff
@@ -19,9 +19,9 @@ installer-help-firefox-release-title = { -brand-name-firefox-browser }
 #   $trackers (number) - number of trackers blocked by Firefox (currently in the thousands)
 installer-help-firefox-release-desc = Get the latest. Automatic privacy is here. Download { -brand-name-firefox-browser } to block over { $trackers } trackers.
 
-installer-help-firefox-beta-title = { -brand-name-firefox-browser } { -brand-name-beta }
+installer-help-firefox-beta-title = { -brand-name-firefox } { -brand-name-beta }
 installer-help-firefox-beta-desc = Test about-to-be released features in the most stable pre-release build.
-installer-help-firefox-developer-title = { -brand-name-firefox-browser } { -brand-name-developer-edition }
+installer-help-firefox-developer-title = { -brand-name-firefox } { -brand-name-developer-edition }
```
Comment (peiying2) [inline en/firefox/installer-help.ftl:24]: add v2 in string ID

File: en/firefox/installer-help.ftl [hunk 2]
```diff
@@ -19,9 +19,9 @@ installer-help-firefox-release-title = { -brand-name-firefox-browser }
 #   $trackers (number) - number of trackers blocked by Firefox (currently in the thousands)
 installer-help-firefox-release-desc = Get the latest. Automatic privacy is here. Download { -brand-name-firefox-browser } to block over { $trackers } trackers.
 
-installer-help-firefox-beta-title = { -brand-name-firefox-browser } { -brand-name-beta }
+installer-help-firefox-beta-title = { -brand-name-firefox } { -brand-name-beta }
```
Comment (peiying2) [inline en/firefox/installer-help.ftl:22]: add v2 in string ID.

File: en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 # HTML page title
 developer-mdnplus-page-title = { -brand-name-firefox-developer-edition }
-developer-mdnplus-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+developer-mdnplus-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox } { -brand-name-developer-edition }.
```
Comment (peiying2) [inline en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:9]: add v2 in string ID.

File: en/firefox/developer.ftl [hunk 4]
```diff
@@ -73,16 +73,26 @@ firefox-developer-firefox-developer-edition-sends = { -brand-name-firefox-develo
 firefox-developer-download-the-firefox-browser = Download the { -brand-name-firefox } browser made for developers
 firefox-developer-welcome-to-the-all-new = Welcome to the all-new { -brand-name-firefox-quantum }: { -brand-name-developer-edition }
 firefox-developer-firefox-has-been-rebuilt = { -brand-name-firefox } has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
-firefox-developer-congrats-you-now-have-firefox = Congrats. You now have { -brand-name-firefox-quantum }: { -brand-name-developer-edition }.
-firefox-developer-this-isnt-just-an-update = This isn’t just an update. This is { -brand-name-firefox-quantum }: A brand new { -brand-name-firefox } that has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
-firefox-developer-welcome-to-firefox-browser = Welcome to { -brand-name-firefox-browser } { -brand-name-developer-edition }
+firefox-developer-welcome-to-firefox-browser = Welcome to { -brand-name-firefox } { -brand-name-developer-edition }
```
Comment (peiying2) [inline en/firefox/developer.ftl:76]: need to add v2 in string ID (matches with line #78 in the older version)

File: en/firefox/developer.ftl [hunk 5]
```diff
@@ -8,7 +8,7 @@
 
 firefox-developer-page-title = { -brand-name-firefox-developer-edition }
 firefox-developer-firefox-developer-edition-desc = { -brand-name-firefox-developer-edition } is the blazing fast browser that offers cutting edge developer tools and latest features like CSS Grid support and framework debugging
-firefox-developer-firefox-browser = { -brand-name-firefox-browser } { -brand-name-developer-edition }
+firefox-developer-firefox-browser = { -brand-name-firefox } { -brand-name-developer-edition }
```
Comment (peiying2) [inline en/firefox/developer.ftl:11]: Need to add v2 in the string ID, I don't think brand name placeholders get automatically updated.

File: en/firefox/developer.ftl [hunk 6]
```diff
@@ -73,16 +73,26 @@ firefox-developer-firefox-developer-edition-sends = { -brand-name-firefox-develo
...
 firefox-developer-all-the-latest-v2 = All the latest developer tools in beta in addition to features like the Multi-line Console Editor and WebSocket Inspector.
-
-# Obsolete string
-firefox-developer-all-the-latest = All the latest developer tools in beta, plus <strong>experimental features</strong> like the Multi-line Console Editor and WebSocket Inspector.
-
 firefox-developer-a-separate-profile = A <strong>separate profile and path</strong> so you can easily run it alongside Release or { -brand-name-beta } { -brand-name-firefox }.
 firefox-developer-preferences-tailored = Preferences <strong>tailored for web developers</strong>: Browser and remote debugging are enabled by default, as are the dark theme and developer toolbar button.
-firefox-developer-congrats-you-now-have = Congrats. You now have { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+firefox-developer-congrats-you-now-have-latest-v2 = You now have the latest version of { -brand-name-firefox } { -brand-name-developer-edition }.
+# Obsolete string (expires: 2024-05-14)
```
Comment (peiying2) [inline en/firefox/developer.ftl:82]: Should we remove this with the new string like how the other string updates in this PR?

File: en/firefox/developer.ftl [hunk 7]
```diff
@@ -73,16 +73,26 @@ firefox-developer-firefox-developer-edition-sends = { -brand-name-firefox-develo
...
 firefox-developer-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+
+
+# Variables:
+#   $attrs (string) - link to the most recent Firefox Developer Edition release notes
+firefox-developer-view-the-release = View the <a { $attrs }>release notes</a> (English only) to see what’s new.
+# this is a link to https://firefox-source-docs.mozilla.org/devtools-user/
+firefox-developer-developer-tools-user = Developer Tools User Docs
```
Comment (peiying2) [inline en/firefox/developer.ftl:90]: where is the link in this string?

Comment (craigcook) [reply en/firefox/developer.ftl:90]: The entire string is wrapped in a link. I can update the comment to try to clarify that.

**Comment by craigcook** [comment]
I'm working on a PR to fix the string IDs, sorry for the mixup.

File: en/firefox/developer.ftl (hunk 7 above)
Comment (peiying2) [reply en/firefox/developer.ftl:90]: Or should we not even mention it?

**Review by peiying2 — APPROVED** [review]
I approve a new commit with legal titles. Right now, these two documents are being localized. Need to make sure the titles here match those in the actual documents.

**Comment by craigcook** [comment]
We shouldn't merge this changeset here until the fixes in https://github.com/mozilla/bedrock/pull/14347 are merged upstream in bedrock (which at this point won't happen until Monday, sorry). That PR amends some of the above strings with different IDs.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Pending fixes from previous commit.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**